docs/Developers/Newcomers_Guide.rst

```diff
@@ -152,7 +186,25 @@ Now you can start working on it.
 
         cobot assign <issue_link>
 
-    Take care to write the full link to the issue
+    Take care to write the full link to the issue.
+    Also take up the issue, only when you know what the problem is and
+    how to solve it.
+
+    You can do amazing stuff using cobot.
+
```

Comment:
I think @supergr35 means the gitter chat-room cobot-test should be mentioned here as a cobot playground, to not spam the main channel while playing with cobot